src/components/pnp/isGoalStepPlannedInsideProject.spec.ts

```diff
@@ -0,0 +1,21 @@
+describe('stepCompleteDate', () => {
+  it('should return null when fiscal quarter is undefined', () => {
```

Comment:
I'd combine these two, they are almost always handled the same way. Only exception is update patches where undefined is missing & null is set to null.
```suggestion
  it('should return null when fiscal quarter is nullish', () => {
```
```suggestion
  it('should return null when fiscal quarter is nil', () => {
```

Reply:
@CarsonF - I waffled on that.  You mean use **either** the _nullish_ or _nil_ verbiage and combine the "undefined" and "null" conditions into one test?

Reply:
Easy to test both within one test as well.